master/pom.xml

```diff
@@ -1648,7 +1661,7 @@
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
```

Comment:
This is in master. Just moved the version to the list of version.  It has been removed in the project